File: PHP/PHP Basics/04.PHP Advanced Syntax Excercises/SumMatrixElements.php
<?php

$matrix = [
    [7, 1, 3, 3, 2, 1],
    [1, 3, 9, 8, 5, 6],
    [4, 6, 7, 9, 1, 0]
];

$sum_of_matrix = 0;

foreach($matrix as $row){
    foreach($row as $col){
       $sum_of_matrix += $col;
    }
}

echo "sum of the matrix: $sum_of_matrix";
?>

File: C# Fundamentals/C# Advanced/03.Multidimensional Arrays/01.SumMatrixElements/SumMatrixElements.cs
using System;
using System.Linq;

namespace _01.SumMatrixElements
{
    public class SumMatrixElements
    {
        public static void Main()
        {
            int[] matrixData = Console.ReadLine()
                .Split(new[] { ',', ' ' }, StringSplitOptions.RemoveEmptyEntries)
                .Select(int.Parse)
                .ToArray();

            int rows = matrixData[0];
            int cols = matrixData[1];

            int[,] matrix = new int[rows, cols];

            for (int row = 0; row < rows; row++)
            {
                int[] colsData = Console.ReadLine()
                    .Split(new[] { ',', ' ' }, StringSplitOptions.RemoveEmptyEntries)
                    .Select(int.Parse)
                    .ToArray();

                for (int col = 0; col < cols; col++)
                {
                    matrix[row, col] = colsData[col];
                }
            }

            int sum = 0;
            for (int row = 0; row < rows; row++)
            {
                for (int col = 0; col < cols; col++)
                {
                    int value = matrix[row, col];
                    sum += value;
                }
            }

            Console.WriteLine(rows);
            Console.WriteLine(cols);
            Console.WriteLine(sum);
        }
    }
}
